src/pages/MapPage.css

```
@@ -15,7 +15,7 @@ h2 {
   font-size: 1.8rem;
   font-weight: 400;
   margin-bottom: 1rem;
-  color: #ffffff;
+  color: #fafafa !important;
```

```suggestion
  color: #ffffff !important;
```
The hex color #fafafa is not white (as suggested in your issue), but #ffffff is white.